src/clinical/api/types.ts

```diff
@@ -15,6 +16,8 @@ export const ClinicalDataApiBody = zod.object({
 });
 export type ClinicalDataApiBody = zod.infer<typeof ClinicalDataApiBody>;
 
+export type ClinicalDataPaginatedBody = ClinicalDataApiBody & PaginationQuery;
```

Comment:
I think we can avoid a messy codebase here as duplication of types is popping up.

https://github.com/icgc-argo/argo-clinical/blob/develop/src/clinical/clinical-service.ts#L72
This seems to be typing the same thing.

`PaginatedClinicalQuery` is still used throughout and

Reply:
Yes this crossed my mind as well, let me review

Reply:
OK, see: https://github.com/icgc-argo/argo-clinical/pull/1139/commits/3cd644374e8a16fff765273fdfc81f52b8d1ca80

I reverted to using only `PaginatedClinicalQuery`, and it turns out `programShortName` was unnecessary to include in this type at all, b/c every function that uses this query is providing programId as the first argument, and query only concerns other details like donorId, entityType, and pagination